testing/idris/APKBUILD

```diff
@@ -18,19 +15,20 @@ source="$pkgname-$pkgver.tar.gz::https://github.com/idris-lang/Idris-dev/archive
 	cabal.config"
 builddir="$srcdir/Idris-dev-$pkgver"
 
-export CABAL_CONFIG="$srcdir/cabal/cabal.config"
+CABAL_HOME="$srcdir/cabal"
+export CABAL_HOME
```

Comment:
Since it's a non-standard variable, could you prefix it with `_`, like `_cabal_home`?